docs/tools/kompose/user-guide.md

```diff
@@ -427,13 +418,17 @@ If you want to create normal pods without controllers you can use `restart` cons
 
 **Note**: controller object could be `deployment` or `replicationcontroller`, etc.
 
-For e.g. `mariadb` service will become pod down here.
+For e.g. `pival` service will become pod down here. This container calculated value of `pi`.
```

Comment:
I'm not sure what the following example is for. Is it an example of how restart is used in docker compose? If so, need more explanation, and also mention what it maps to in Kubernetes concept.